docs/hstaxe/axe_tasks.rst

```diff
@@ -249,7 +251,7 @@ AXECORE
 [AXECORE] This aXe task combines the Low Level Tasks sex2gol, gol2af,
 af2pet, petff, petcont, pet2spc and stamps and offers the possibility to
 make a complete aXe reduction based on the individual images in one
-task. This also includes the reduction with background PETs (set
+task. This also includes local background subtraction (set
```

Comment:
I feel like this is a repeat of my constant `dispersion` vs `distortion` flubs. I want to TRIPLE check that you're referring to `local` subtraction and not `global` subtraction here in context of setting `back=YES`

Reply:
Yes, AXEPREP does global subtractrion, AXECORE does local subtraction.